test/contracts/root-element.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import React = require('react');
+import {findDOMNode} from 'react-dom';
+import {ClientRenderer, expect, RenderingContext} from 'test-drive-react';
+import * as WixReactComponents from '../../src';
+import {isReactComponent} from '../utils/is-react-component';
+
+// const allComponents = Object.keys(WixReactComponents);
+// Not all components are passing the contract test right now.
+// failing: TreeView, NumberInput, Toggle, BirthdayPicker, RadioButton, RadioGroup, Portal
+
+const testedComponents: string[] = ['DatePicker', 'CheckBox', 'Image', 'DropDown'];
+
+describe('Root Element contract', function() {
```

Comment:
The module contains both contract test and spec running over (potentially all) components